src/main/java/seedu/address/MainApp.java

```diff
@@ -15,19 +15,10 @@
 import seedu.address.commons.util.StringUtil;
 import seedu.address.logic.Logic;
 import seedu.address.logic.LogicManager;
-import seedu.address.model.Model;
-import seedu.address.model.ModelManager;
-import seedu.address.model.ReadOnlyStudentList;
-import seedu.address.model.ReadOnlyUserPrefs;
-import seedu.address.model.StudentList;
-import seedu.address.model.UserPrefs;
+import seedu.address.model.*;
```

Comment:
Change back to the previous import statements? Need to list out all the imported classes instead of using *